core/src/main/java/org/mule/runtime/core/internal/event/AbstractEventContext.java

```diff
@@ -220,8 +219,8 @@ protected void tryComplete() {
 
     synchronized (this) {
       if (state == STATE_RESPONSE && allChildrenComplete) {
-        if (LOGGER.isDebugEnabled()) {
-          LOGGER.debug(this + " completed.");
+        if (debugLogEnabled) {
+          LOGGER.debug("{} completed.", this);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Invoke method(s) only conditionally. ([squid:S2629](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2629))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7701&issues=AWkLjRWWotIXnmsUmgk4&open=AWkLjRWWotIXnmsUmgk4)

[ISSUE_KEY]: AWkLjRWWotIXnmsUmgk4